runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -237,6 +244,48 @@ public void onChatMessage(ChatMessage chatMessage)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick event)
```

Comment:
So this is just to delay the loading 1 game tick? I am starting to feel like we do this way too often we should probably have some util method for it, like scheduleLater or something, maybe :d